### PR TITLE
[new release] swapfs (0.1.0)

### PR DESCRIPTION
**Pull request by reynir**
Streaming temporary storage for Mirage OS

- Project page: <a href="https://github.com/robur-coop/mirage-swapfs">https://github.com/robur-coop/mirage-swapfs</a>
- Documentation: <a href="https://robur-coop.github.io/mirage-swapfs/doc">https://robur-coop.github.io/mirage-swapfs/doc</a>

##### CHANGES:

* First release! 🥳

### CODE REVIEW

File: packages/swapfs/swapfs.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+maintainer: ["Robur Team <team@robur.coop>"]
+authors: ["Reynir Björnsson <reynir@reynir.dk>"]
+license: "MIT"
+tags: ["org:mirage" "org:robur" "swap"]
+homepage: "https://github.com/robur-coop/mirage-swapfs"
+doc: "https://robur-coop.github.io/mirage-swapfs/doc"
+bug-reports: "https://github.com/robur-coop/mirage-swapfs/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.16" & >= "3.16.0"}
```
[the remark this answers is not in the document]
Comment (raphael-proust) [reply packages/swapfs/swapfs.0.1.0/opam:18]: Maybe look into changing this upstream. Possibly also `dune` should warn about this when generating opam packages from `dune-project` files (assuming that's where it comes from).

[the remark this answers is not in the document]
Comment (reynir) [reply packages/swapfs/swapfs.0.1.0/opam:18]: Ohhh! Indeed. I guess I was a bit too eager when adding version constraints in dune-project. I'll open an issue with dune.

[the remark this answers is not in the document]
Comment (reynir) [reply packages/swapfs/swapfs.0.1.0/opam]: https://github.com/ocaml/dune/issues/11106

**Comment by reynir** [comment]
I think this should be good to merge now - the previous CI run passed fine, and the latest change should not affect that.

**Comment by mseri** [comment]
Thanks. Don't forget to announce this on discuss.ocaml.org if you hadn't already done it

**Pull request merged**